launcher/internal/experiments/experiments_test.go

```diff
@@ -0,0 +1,51 @@
+package experiments
+
+import (
+	"testing"
+)
+
```

Comment:
Add a test for empty string and empty JSON "{}".

Reply:
Added the empty string, the empty bracket is in the next test case.